Kwf/Controller/Action/Auto/Grid.php

```diff
@@ -1010,15 +1010,15 @@ public function downloadExportFileAction()
         if (!isset($this->_permissions['xls']) || !$this->_permissions['xls']) {
             throw new Kwf_Exception("XLS is not allowed.");
         }
-        if (!file_exists('temp/'.$this->_getParam('downloadkey').'.xls')) {
+        if (!file_exists('temp/'.$this->_getParam('downloadkey').'.xlsx')) {
             throw new Kwf_Exception('Wrong downloadkey submitted');
         }
         Kwf_Util_TempCleaner::clean();
 
         $file = array(
-            'contents' => file_get_contents('temp/'.$this->_getParam('downloadkey').'.xls'),
-            'mimeType' => 'application/msexcel',
-            'downloadFilename' => 'export_'.date('Ymd-Hi').'.xls'
+            'contents' => file_get_contents('temp/'.$this->_getParam('downloadkey').'.xlsx'),
+            'mimeType' => 'application/octet-stream',
```

Comment:
is this the correct mime type for xlsx?
I think it is this one: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
References:
http://www.sitepoint.com/web-foundations/mime-types-summary-list/
http://stackoverflow.com/questions/4212861/what-is-a-correct-mime-type-for-docx-pptx-etc